package.json

```diff
@@ -210,11 +208,10 @@
     "punycode": "1.4.1",
     "qrcode": "1.4.4",
     "qs": "6.9.7",
-    "querystring-es3": "0.2.1",
+    "querystring-es3": "~0.2.0",
```

Comment:
any reason to not pin to a specific version here?

Reply:
not even sure how that happened, i didn't change it (or use ai) 🤔 good catch